tests/ci_build/build_jvm_packages.sh

```diff
@@ -40,8 +37,12 @@ if [ "x$gpu_options" == "x" ] && [ "x$use_scala213" == "x" ]; then
   mvn --no-transfer-progress clean package -Dspark.version=3.4.3 -pl xgboost4j,xgboost4j-spark
 fi
 
-mvn --no-transfer-progress clean package -Dspark.version=${spark_version} $gpu_options
+mvn --no-transfer-progress clean install -Dspark.version=${spark_version} $gpu_options
 
+# Integration tests
```

Comment:
True, I will co-work with jiaming to make GPU support array, then we can enable this tests for GPU as well.

Reply:
Do we have separate demos for the GPU package? Right now all demos in `xgboost4j-example` are CPU-only.

Reply:
Yeah, I plan to add it for GPU package, but should wait for the PR supporting array/vector input in GPU.